### PR TITLE
Gpl 957 sentinel sample creation response

### LINKED ISSUE #65: GPL-957 BUG - Create Sentinel Samples no response
**Describe the bug**
On creating samples via Sample Creation page,there is no feedback if successful

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #65 

Changes proposed in this pull request:

*Alert refactor for sentinel sample creation page
*
*

### CODE REVIEW

File: pages/sentinel_create_samples.vue [hunk 1]
```diff
@@ -95,17 +95,20 @@ export default {
...
         const msg = errored.map((e) => e.errors.join(', ')).join(', ')
-        this.alertMessage = msg
-        this.showDismissibleAlert = true
+        this.showAlert(msg, 'danger')
       }
 
       const successful = resp.filter((obj) => Object.keys(obj).includes('data'))
       if (successful.length > 0) {
         this.items = successful.map((obj) => obj.data).map((obj) => obj.data)
+        if (!errored.length) { this.showAlert('Sentinel samples successfully created in sequencescape', 'success') }
```
Comment (harrietc52) [inline pages/sentinel_create_samples.vue:104]: Did we want to add an warning if it is a partical success saying the items in the table have been sucessfully added to SS, and these are the barcodes that have failed?

**Review by harrietc52 — DISMISSED** [review]
Looks great 👍

File: pages/sentinel_create_samples.vue [hunk 2]
```diff
@@ -95,17 +95,20 @@ export default {
       const errored = resp.filter((obj) => Object.keys(obj).includes('errors'))
       if (errored.length > 0) {
         const msg = errored.map((e) => e.errors.join(', ')).join(', ')
-        this.alertMessage = msg
-        this.showDismissibleAlert = true
+        this.showAlert(msg, 'danger')
```
Comment (harrietc52) [inline pages/sentinel_create_samples.vue:98]: Maybe here doing a check if sucessful.length > 0 then showAlert with 'warning' type and a better error message

Comment (harrietc52) [reply pages/sentinel_create_samples.vue:98]: Might also be a cleaner way to handle all this logic. 
- errors no success
- errors and success
- success no errors

File: pages/sentinel_create_samples.vue [hunk 3]
```diff
@@ -99,15 +99,15 @@ export default {
...
+      this.sentinelSampleCreationAlert(errored, successful)
     },
-    sampleCreationAlert(errored, successful) {
+    sentinelSampleCreationAlert(errored, successful) {
       const msg = errored.map((e) => e.errors.join(', ')).join(', ')
-      if (errored.length && successful.length) {
+      if (errored.length > 0 && successful.length > 0) {
         this.showAlert(`Some samples were successfully created however: ${msg}`, 'warning')
-      } else if (errored.length) {
+      } else if (errored.length > 0) {
```
Comment (harrietc52) [inline pages/sentinel_create_samples.vue:108]: `} else if (errored.length > 0 && successful.length === 0) {`

**Review by harrietc52 — APPROVED** [review]
minor suggestion

File: pages/sentinel_create_samples.vue (hunk 3 above)
Comment (BenTopping) [reply pages/sentinel_create_samples.vue:108]: it should never get to this if case if successful.length does not equal 0 as it is an else if block but i could add that to make it more explicit

**Review by harrietc52 — APPROVED** [review]

**Pull request merged**